### DISCUSSION TITLE
Confuse to use G-helper for gaming with my newly purchased rog zephyrus duo 16 (2023)

### DISCUSSION THREAD

**Discussion by smithpoon** [seerge/g-helper#987] [Q&A]
**Is your feature request related to a problem? Please describe.**
I recently acquired a new rog zephyrus duo 16 (2023) laptop, and recently started using g-helper becasue the AC is complicated to understand, there is nothing unusual for normal using with cpu temperature may got below 70 degree celcius. But when I modify the setting for 3A gaming, the cpu soon exceeds the temperature wall and started functioning improperly.

**Describe the solution you'd like**
Would like to seek what kind of gpu and cpu setting for gaming, especially for 3A games with best resolution.

**Comment by seerge** [comment]
Moved this to discussions as this is not an "issue". It's absolutely up to you how to configure your laptop, default modes baked in bios work absolutely fine usually.
